doc/ref_model/chapters/chapter06.rst

```diff
@@ -241,9 +517,24 @@ Intra-Cloud Infrastructure Interfaces
 Hypervisor Hardware Interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Table 6-1 lists a number of NFVI and VIM interfaces, including the internal VI-Ha interface. The VI-Ha interface allows the hypervisor to control the physical infrastructure; the hypervisor acts under VIM control. The VIM issues all requests and responses using the NF-VI interface; requests and responses include commands, configuration requests, policies, updates, alerts, and response to infrastructure results. The hypervisor also provides information about the health of the physical infrastructure resources to the VM. All these activities, on behalf of the VIM, are performed by the hypervisor using the VI-Ha interface. While no abstract APIs have yet been defined for this internal VI-Ha interface, ETSI GS NFV-INF 004 [9] defines a set of requirements and details of the information that is required by the VIM from the physical infrastructure resources. Hypervisors utilize various programs to get this data including BIOS, IPMI, PCI, I/O Adapters/Drivers, etc.
+Table 6-1 lists a number of NFVI and VIM interfaces, including the internal VI-Ha interface. The VI-Ha interface allows
+the hypervisor to control the physical infrastructure; the hypervisor acts under VIM control. The VIM issues all
+requests and responses using the NF-VI interface; requests and responses include commands, configuration requests,
+policies, updates, alerts, and response to infrastructure results. The hypervisor also provides information about the
+health of the physical infrastructure resources to the VM. All these activities, on behalf of the VIM, are performed by
+the hypervisor using the VI-Ha interface. While no abstract APIs have yet been defined for this internal VI-Ha
+interface, ETSI GS NFV-INF 004 [9] defines a set of requirements and details of the information that is required by the
+VIM from the physical infrastructure resources. Hypervisors utilize various programs to get this data including BIOS,
+IPMI, PCI, I/O Adapters/Drivers, etc.
 
 Enabler Services Interfaces
 ---------------------------
 
+<<<<<<< HEAD
+An operational cloud needs a set of standard services to function. Services such as NTP for time synchronization, DHCP
+for IP address allocation, DNS for obtaining IP addresses for domain names, and LBaaS (version 2) to distribute incoming
+SSrequests amongst a pool of designated resources.
+
+=======
 An operational cloud needs a set of standard services to function. Services such as NTP for time synchronization, DHCP for IP address allocation, DNS for obtaining IP addresses for domain names, and LBaaS (version 2) to distribute incoming requests amongst a pool of designated resources.
+>>>>>>> master
```

Comment:
something went wrong here

Reply:
Uh-oh, that is an unhandled merge conflict.

Reply:
Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)